ModelBundle/Resources/config/validation/node.yml

```diff
@@ -2,6 +2,7 @@ OpenOrchestra\ModelBundle\Document\Node:
     constraints:
         - OpenOrchestra\ModelBundle\Validator\Constraints\PreventProhibitedStatusChange: ~
         - OpenOrchestra\ModelBundle\Validator\Constraints\CheckRoutePattern: ~
+        - OpenOrchestra\ModelBundle\Validator\Constraints\UniqueNodeOrder: ~
```

Comment:
The constraint can't be satisfy on move action cause we override every nodes "order" attribute and flush them together.